docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -5,60 +5,48 @@ Here, we describe several ways to access data.
 
 Broadly speaking, there are three kinds of ScPCA data you might wish to work with:
 
-
-- Data from the ScPCA Portal
-    - You can find out more about the contents of files and how they were processed from the ScPCA documentation: <https://scpca.readthedocs.io>
-- Results from other OpenScPCA modules
-    - You can obtain results from modules that have been added to the `OpenScPCA-nf` workflow by [following the instructions below](#accessing-scpca-module-results).
+1. Data from the ScPCA Portal
+    - You can find out more about the contents of files and how they were processed from the ScPCA documentation: <https://scpca.readthedocs.io>.
+    - OpenScPCA project contributors [with an Amazon Web Services (AWS) account](./index.md#getting-access-to-aws) can access data using the provided `download-data.py` script, [as described below](#using-the-download-data-script).
+        - If you have not yet been onboarded to OpenScPCA, you will need to access data [directly from the ScPCA Portal](#accessing-data-from-the-scpca-portal).
```

Comment:
I don't think this should be a subpoint, and I think it should be phrased positively.
```suggestion
    - All data is also available [directly from the ScPCA Portal](#accessing-data-from-the-scpca-portal) for users who have not yet fully joined the OpenSCPCA project.
```